### DISCUSSION TITLE
Icons are black

### DISCUSSION THREAD

**Discussion by NitinIsFit** [community/community#28238] [Mobile] [answered]
Please make them yellow and blue, and more contrasting.

![Screenshot_20220803-235848.png](https://user-images.githubusercontent.com/94877880/182682942-5618bcb4-9d20-446c-82cb-67d8653e62f2.png)

**Comment by stevepopovich** [comment, marked as answer]
Sorry, which icons are you referring to?

**Reply by radar-blip** [reply]
Icons in front of 'Note' and 'Warning' they were originally blue and gold-ish. They probably would have been ok just left alone.
These:
![notes](https://user-images.githubusercontent.com/39196767/182696680-df23ec1d-b448-49bf-864d-a92c7387d577.png)

**Reply by stevepopovich** [reply]
Gotcha, I will file an issue for prioritization. Thanks for reaching out!

**Reply by stevepopovich** [reply]
A fix is actually coming for this in the next release! Thanks for reporting!

**Reply by NitinIsFit** [reply]
> A fix is actually coming for this in the next release! Thanks for reporting!

Thanks, appreciated! 🎉
